modules/bliinkBidAdapter.js

```diff
@@ -180,37 +209,58 @@ export const isBidRequestValid = (bid) => {
  */
 export const buildRequests = (validBidRequests, bidderRequest) => {
   if (!validBidRequests || !bidderRequest || !bidderRequest.bids) return null
-
+  const domLoadingDuration = getDomLoadingDuration().toString();
   const tags = bidderRequest.bids.map((bid) => {
+    let bidFloor;
+    const sizes = bid.sizes.map((size) => ({ w: size[0], h: size[1] }));
+    const mediaTypes = Object.keys(bid.mediaTypes)
+    if (typeof bid.getFloor === 'function') {
+      bidFloor = bid.getFloor({
+        currency: CURRENCY,
+        mediaType: mediaTypes[0],
+        size: sizes[0]
```

Comment:
This is fine, but will not support [multi format bid requests ](https://docs.prebid.org/dev-docs/show-multi-format-ads.html#:~:text=Prebid%20multi%2Dformat%20ad%20units,formats%20and%20their%20respective%20properties.)